app/src/ui/app.tsx

```diff
@@ -2338,9 +2338,9 @@ export class App extends React.Component<IAppProps, IAppState> {
     const { currentDragElement } = this.state
     const isCherryPickCommitBeingDragged =
```

Comment:
Just saw that I missed renaming `isCherryPickCommitBeingDragged` to not be cherry-pick specific. But, changing this line in next PR anyway so I will catch it there